eraser-map.yaml

```diff
@@ -794,6 +794,16 @@ styles:
         base: points
         texture: pois
         interactive: true
+        
+    icons-overlay:
+        base: points
+        texture: pois
+        interactive: true
```

Comment:
In this style and the one above it, `interactive` is not a valid parameter. `interactive` is a parameter for draw rules, on styles it will do nothing. (see the [docs](https://github.com/tangrams/tangram-docs/blob/gh-pages/pages/draw.md#interactive))